src/components/player/index.vue

```diff
@@ -461,6 +461,16 @@ export default {
 		},
 		loggedIn() {
 			this.buildTray();
+		},
+		radioType() {
+			this.buildTray();
+			if (this.playing) {
+				this.togglePlaying();
+				this.togglePlaying();
+			}
+		},
+		playing() {
+			this.buildTray();
```

Comment:
Do we have to rebuild the tray everytime a watchable changes? I've been doing the same thing, but it seems inefficient.

Reply:
I don't know, as I've not worked with trays before.